src/pages/home/report/ContextMenu/ContextMenuActions.tsx

```diff
@@ -200,7 +201,9 @@ const ContextMenuActions: ContextMenuAction[] = [
         onPress: (closePopover, {reportAction, reportID}) => {
             if (closePopover) {
                 hideContextMenu(false, () => {
-                    ReportActionComposeFocusManager.focus();
+                    InteractionManager.runAfterInteractions(() => {
+                        ReportActionComposeFocusManager.focus(true);
```

Comment:
Please add a comment here to explain why we are passing `true`.

Reply:
added but can u check the comments.

Reply:
Let's avoid creating platform specific language.

Can you answer this question without referencing "native":

What behavior are we giving the callback by passing `true`?